packages/discovery-provider/src/api/v1/users.py

```diff
@@ -652,11 +652,10 @@ def get(self, handle, authed_user_id):
 
 USER_REPOSTS_ROUTE = "/<string:id>/reposts"
 
-reposts_response = make_response(
-    "reposts", ns, fields.List(fields.Nested(activity_model))
-)
+reposts_response = make_response("reposts", ns, fields.List(activity_model))
+
 full_reposts_response = make_full_response(
-    "full_reposts", full_ns, fields.List(fields.Nested(activity_model_full))
+    "full_reposts", full_ns, fields.List(activity_full_model)
```

Comment:
This no longer needs `fields.Nested` because `activity_full_model` is a Polymorph now right?

Reply:
oh whoops actually I think it might need it and I just missed this.
I will figure out the correct implementation and make sure both default/full are using the same pattern